### PR TITLE
Specification Change and Regenerated for Azure Stack Fabric Admin

### PR DESCRIPTION
**Pull request by deathly809**
## Description

The underlying spec was rewritten.

---

This checklist is used to make sure that common guidelines for a pull request are followed.
- [X] Please add REST spec PR link to the SDK PR
- [X] **I have read the [contribution guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md).**
- [ ] **The pull request does not introduce [breaking changes](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/Documentation/breaking-changes.md).**

### [General Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md#general-guidelines)
- [X] Title of the pull request is clear and informative.
- [X] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For more information on cleaning up the commits in your PR, [see this page](https://github.com/Azure/azure-powershell/blob/dev/documentation/cleaning-up-commits.md).

### [Testing Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md#testing-guidelines)
- [X] Pull request includes test coverage for the included changes.

### [SDK Generation Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md#sdk-generation-guidelines)
- [X] If an SDK is being regenerated based on a new swagger spec, a link to the pull request containing these swagger spec changes has been included above.
- [X] The generate.cmd file for the SDK has been updated with the version of AutoRest, as well as the commitid of your swagger spec or link to the swagger spec, used to generate the code.
- [X] The `*.csproj` and `AssemblyInfo.cs` files have been updated with the new version of the SDK.

### CODE REVIEW

File: src/AzureStack/FabricAdmin/Fabric.Admin/Microsoft.AzureStack.Management.Fabric.Admin.csproj [hunk 1]
```diff
@@ -1,9 +1,6 @@
 ﻿
 <Project Sdk="Microsoft.NET.Sdk">
-    <!-- Please do not move/edit code below this line -->
-    <Import Project="$([MSBuild]::GetPathOfFileAbove('AzSdk.reference.props'))" />
-    <!-- Please do not move/edit code below this line -->
-
+  <Import Project="$([MSBuild]::GetPathOfFileAbove('AzSdk.reference.props'))" />
```
Comment (dsgouda) [inline src/AzureStack/FabricAdmin/Fabric.Admin/Microsoft.AzureStack.Management.Fabric.Admin.csproj]: Looks like these are breaking changes. Though `0.1.1-preview` version has been published, please consider bumping the minor version (`0.2.0-preview`)

**Review by dsgouda — CHANGES_REQUESTED** [review]
Apart from the version number, looks ok

File: src/AzureStack/FabricAdmin/Fabric.Admin/Microsoft.AzureStack.Management.Fabric.Admin.csproj [hunk 2]
```diff
@@ -1,13 +1,10 @@
...
   <PropertyGroup>
     <PackageId>Microsoft.AzureStack.Management.Fabric.Admin</PackageId>
     <Description>Provides developers with libraries for the fabric admin under AzureStack to manage fabric infrastructure.</Description>
-    <VersionPrefix>0.1.1-preview</VersionPrefix>
+    <VersionPrefix>0.2.0-preview</VersionPrefix>
     <AssemblyName>Microsoft.AzureStack.Management.Fabric.Admin</AssemblyName>
```
Comment (dsgouda) [inline src/AzureStack/FabricAdmin/Fabric.Admin/Microsoft.AzureStack.Management.Fabric.Admin.csproj]: Please add [these](https://github.com/Azure/azure-sdk-for-net/blob/psSdkJson6/src/SDKs/Compute/Management.Compute/Microsoft.Azure.Management.Compute.csproj#L23) lines at the bottom of the csproj

**Review by dsgouda — APPROVED** [review]
Looks great apart from a minor change

File: src/AzureStack/FabricAdmin/Fabric.Admin/Microsoft.AzureStack.Management.Fabric.Admin.csproj (hunk 2 above)
Comment (deathly809) [reply src/AzureStack/FabricAdmin/Fabric.Admin/Microsoft.AzureStack.Management.Fabric.Admin.csproj]: Done.

**Comment by deathly809** [comment]
@dsgouda Everything look good?

**Pull request merged**